package.json

```diff
@@ -40,66 +41,59 @@
     ],
     "delay": "2500"
   },
-  "husky": {
-    "hooks": {
-      "pre-commit": "lint-staged",
-      "pre-push": "npm run verify"
-    }
-  },
   "jest": {
     "verbose": true,
     "testMatch": ["**/__tests__/**/*.[jt]s?(x)"]
   },
   "dependencies": {
-    "@sentry/node": "^5.29.0",
+    "@sentry/node": "^6.3.1",
```

Comment:
"Starting from the version 6.0.0, all SDKs that support sending sessions data will do so by default."  [link](https://docs.sentry.io/product/releases/health/)